### ISSUE TITLE
difftest with multiple cores, using tiles

### ISSUE THREAD

**Issue by joamatab** [gdsfactory/gdsfactory#3038]
How can we make a tiled version of difftest to leverage multiple cores and using tiles?

for big layouts current difftest

**Comment by tvt173** [comment]
i don't believe this is relevant anymore... tests now run with multiple cores using pytest-xdist

**Comment by sebastian-goeldi** [comment]
nto very useful tbh. Can be done, but painful to implement, closing as not planned
